git_machete/client.py

```diff
@@ -2362,3 +2375,21 @@ def __sync_before_creating_pr(self, *, opt_onto: Optional[LocalBranchShortName],
             if ans in ('y', 'yes'):
                 return
             raise MacheteException('Pull request creation interrupted.')
+
+    def remove_untracked_branches(self,
```

Comment:
Rel (maybe to be shipped together with this PR): https://github.com/VirtusLab/git-machete/issues/98